Makefile

```diff
@@ -4,10 +4,10 @@ build:
 
 .PHONY: sql
 sql:
-	C:/sqlite/sqlite3 ./db/finance.db < createdb.sql
+	C:/sqlite/sqlite3 ./db/finance.db < createdb.sql # Version of linux to run it on ubuntu :)
```

Comment:
Because other people don't like we change their code.
I just propose an idea.
As you want, I gonna write it :)